clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/ClioIndex.scala

```diff
@@ -64,6 +66,7 @@ trait ClioIndex {
     implicit val mt: ClassTag[MetadataType] = metadataTag
     implicit val qit: ClassTag[QueryInputType] = queryInputTag
     implicit val ke: Encoder[KeyType] = keyEncoder
+    implicit val oke: ObjectEncoder[KeyType] = objectKeyEncoder
```

Comment:
Is `ObjectEncoder <: KeyEncoder`? If so we can get rid of the existing `keyEncoder` field + implicit.